kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -24,9 +24,10 @@
 
         <div class="user-field">
           <label for="user-role"><span class="visuallyhidden">User Role</span></label>
-          <select v-model="role_new" id="user-role">
-            <option :selected="role_new == learner" v-if="role_new" value="learner"> Learner </option>
-            <option :selected="role_new == admin" value="admin"> Admin </option>
+          <select v-model="kind_new" id="user-role">
+            <option :value="LEARNER"> Learner </option>
+            <option :value="ADMIN"> Admin </option>
+            <option :value="COACH"> Coach </option>
```

Comment:
I'm pretty sure `:selected` works for vue2. `:` is the shorthand for `v-bind`. the error you see before the change came from `"role_new == learner"` where `learner` is undefined, and it need to be a string. So `:selected="role_new == 'learner'"` will fix the error.